packages/enzyme-adapter-react-16/package.json

```diff
@@ -33,7 +33,7 @@
     "enzyme-adapter-utils": "2.9.1",
     "lodash": "^4.17.4",
     "object.values": "^1.0.4",
-    "prop-types": "^15.5.10"
+    "prop-types": "^16.0.0-0"
```

Comment:
I don't think there's been a 16 release for `prop-types`. I also don't think there's any intention to keep the `prop-types` package in sync with React's versioning, so React 16 will still be using `prop-types` 15.